xilem_web/web_examples/async_repeat/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Async repeat example | Xilem Web</title>
```

Comment:
I wonder if we should call the example different (though it's most descriptive for the view at least), like interval or ping or (news-)ticker as was suggested [here](https://github.com/linebender/xilem/pull/403#discussion_r1650740496), and maybe add the alternative (`interval`) there as well, so that people don't come to the idea to use `async_repeat`, when their use case is `interval`. At the very least, it should mention `interval` for this use-case.

Reply:
I renamed it to `spawn_tasks`, but that's probably not really appropriate either :thinking: